src/constants/localStorage.ts

```diff
@@ -13,6 +13,7 @@ export enum LocalStorageKey {
   PushNotificationsEnabled = 'dydx.PushNotificationsEnabled',
   PushNotificationsLastUpdated = 'dydx.PushNotificationsLastUpdated',
   TransferNotifications = 'dydx.TransferNotifications',
+  TriggerOrderNotifications = 'dydx.TriggerOrderNotifications',
```

Comment:
mm do we need to store these in localstorage? (do they need to persist across sessions?)

Reply:
hmm, it's probably not important for them to persist 🤔 lemme double check with product

Reply:
hmm actually confirmed with design that we do want them to persist (would want users to see it in their notif bar even if they accidentally closed it). We could set an expiration date for them, but that's not currently supported by the framework so will probably defer looking into that unless feedback comes in

Reply:
i meant across sessions as in after user closes the window! i don't think we need to save those notifications right? they can probably still be in the notification bar without saving to local storage?

Reply:
hmm so the notification bar unfortunately does seem to only read from local storage (it calls [useNotifications](https://github.com/dydxprotocol/v4-web/blob/main/src/views/menus/NotificationsMenu.tsx#L50) which gets its notifications from [localStorage](https://github.com/dydxprotocol/v4-web/blob/a8f976c62cf987c23ac23d43a0b2336ab20cedd2/src/hooks/useNotifications.tsx#L49) (and slowly filters them down)). Are you concerned about the pop ups coming up after the user opens a new window if they haven't dismissed them?